### ISSUE TITLE
Commit For FDA Files

### ISSUE THREAD

**Issue by FurkyT** [marksgraham/OCT-Converter#84]
Hello, 
I was working on with this repo for fda files and I can not extract the images with decode function after that I add a try except and with except part I decode the data with bytes io. And I add some functions to get the other informations in fda file and wrote another .py file with a function that extract bscans images, video, fundus image, gray scale fundus image and a json which contains other informations in fda file.

**Comment by marksgraham** [comment]
Hi @FurkyT ,

Thanks for the PR. I'll try to get round to reviewing soon but I've a couple of questions:

What FDA file were you getting the decode problems with? Personal ones of the general biobank one I link to in the example? What exactly was the error?

Also curious to know where you learned the header structures for all these other data items from? Are they documented somewhere?

**Comment by FurkyT** [comment]
Thanks for reviewing soon,

For the first question I got the error from a fda file that I need to extract for my work but after that I tried the biobank one but nothing change. The error vas a Value Error from pylibjpeg decode function "raise ValueError("Unable to decode the data")"
after I got the error for example and my work file I thought there must be some problem with decode function from that package and I tried several other decoder but the other decoders did not work either than I used the bytes io.

For the second question all of the other header structures from https://bitbucket.org/uocte/uocte/wiki/Topcon%20File%20Format this website I wrote the structures and functions one by one and try to tested it with my limited knowledge about OCT data (but the most of the function outputs has a meaningful data) except the patient info one because the website has structures for @PATIENT_INFO_02 chunk but I had  @PATIENT_INFO_03 but I wrote a function and header structer for @PATIENT_INFO_02 which has information on the website. I just try to get every information that can be taken from a fda file.
